src/Table.php

```diff
@@ -489,7 +489,7 @@ public function renderView()
         $this->_rendered_rows_count = 0;
         foreach ($this->model as $this->current_id => $tmp) {
             $this->current_row = $this->model->get();
-            if ($this->hook('beforeRow') === false) {
+            if ($this->hook(self::HOOK_BEFORE_ROW) === false) {
```

Comment:
see - here you use `self::HOOK_BEFORE_ROW`, but in other places (classes) used like
`$table->onHook(Lister::HOOK_BEFORE_ROW, ...)`